test/integration/mpi_job_controller_test.go

```diff
@@ -740,6 +866,19 @@ func updatePodsToPhase(ctx context.Context, client kubernetes.Interface, pods []
 	return nil
 }
 
+func updatePodsCondition(ctx context.Context, t *testing.T, client kubernetes.Interface, pods []corev1.Pod, condition corev1.PodCondition) error {
```

Comment:
```suggestion
func updatePodsCondition(ctx context.Context,, client kubernetes.Interface, pods []corev1.Pod, condition corev1.PodCondition) error {
```

Do we need to receive `*testing.T`? I don't think so since this function is guaranteed that the external package can not access.

Reply:
Yes, of course, it is only forgotten debug.